src/SELF_DGModel3D_t.f90

```diff
@@ -271,8 +269,7 @@ subroutine CalculateSolutionGradient_DGModel3D_t(this)
 
     ! perform the side exchange to populate the
     ! solutionGradient % extBoundary attribute
-    call this%solutionGradient%SideExchange(this%mesh, &
-                                            this%decomp)
+    call this%solutionGradient%SideExchange(this%mesh)
```

Comment:
Need to double check that the gradient calculations are enabled in the advection diffusion 3d test runs.

Reply:
Resolved in [7c5a7cd](https://github.com/FluidNumerics/SELF/pull/53/commits/7c5a7cd1afac570830df40de4119b3cde91d1012)